addons/binding/org.openhab.binding.solarlog/README.md

```diff
@@ -0,0 +1,24 @@
+# openHAB Binding for SolarLog 
+
+The [SolarLog Family](http://www.solar-log.com/en/home.html) of monitoring devices for PV installations provide a MODBUS (TCP) and JSON-based API to access a number of internal data related to power generation and consumption. 
+
+This binding is an attempt to simplify the reading of these values to be displayed in openHAB. It was born out of the "frustration" that setting up SolarLog monitoring via MODBUS is almost as tedious as writing an entirely new (albeit simple) openHAB binding. And using the HTTP binding is not an option since the JSON interface requires a POST with a payload, which the HTTP binding so far does not support.
+
+## Use of the binding
+
+The SolarLog is exposed as one thing with a number of channels that can be used to read the values for different aspects of your SolarLog installation. 
+
+## Setup of the binding
+
+You can either configure the Thing via the openHAB UI or via a `*.things` file, according to the following example:
+
+`Thing solarlog:datalogger:pv "SolarLog 300" @ "Utility Room" [ url="http://solar-log" refreshInterval="15" ]`
```

Comment:
refreshInterval is defined as an integer, so the correct configuration should be `refreshInterval=15`.

Btw, the template comes with a "Full Example" section, that we have in all readmes of the bindings. It should not only show an example Thing, but also an items and possibly a sitemap file.

Additionally, the readme lacks a table of all the available channels. If users want to use textual configuration, they are currently lost here.